### PR TITLE
chore: integrate codespell and fix existing spelling issues

### PR DESCRIPTION
**Pull request by jviotti**
This tool scans the comments of every source file and reports back
spelling errors.

Signed-off-by: Juan Cruz Viotti <jviotti@openmailbox.org>

### CODE REVIEW

File: lib/gui/scss/modules/_bootstrap.scss [hunk 1]
```diff
@@ -26,7 +26,7 @@ body {
   background-color: $palette-theme-dark-background;
 }
 
-// Fix slighly checkbox vertical alignment issue
+// Fix slightly checkbox vertical alignment issue
```
Comment (lurch) [inline lib/gui/scss/modules/_bootstrap.scss]: This should probably be 'slight' rather than 'slightly'

**Comment by lurch** [comment]
IMHO the commit-message should clarify that this PR _also_ corrects the initial errors found by `codespell` (unless you wanted to split that off into a separate 'typo' PR?).

File: lib/gui/scss/modules/_bootstrap.scss (hunk 1 above)
Comment (jviotti) [inline lib/gui/scss/modules/_bootstrap.scss]: Done!

**Comment by jviotti** [comment]
@lurch Clarified!

**Comment by lurch** [comment]
Some of the Travis builds failed because `pip` was missing https://travis-ci.org/resin-io/etcher/builds/196958253

Impossible to tell if that's been fixed in a newer commit, because Travis seems to have got "stuck" somewhere :-(  https://travis-ci.org/resin-io/etcher/builds
Possibly related to the huge [OSX Backlog](https://www.traviscistatus.com/)  (again :cry: )

**Comment by jviotti** [comment]
Yeah, I've added to the Dockerfiles, so I guess it should work now, but let's wait for Travis to recover :)

**Pull request merged**